kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/DerivedResourcesTest.java

```diff
@@ -65,9 +67,10 @@
 
 class DerivedResourcesTest {
 
-    static final YAMLMapper YAML_MAPPER = new YAMLMapper()
+    static final ObjectMapper YAML_MAPPER = new YAMLMapper()
```

Comment:
Why the LHS type switch? Curiosity rather than a problem

Reply:
`registerModule` returns ObjectMapper, so options were introducing a method or switching to ObjectMapper